packages/plugin-javascript/src/utilities.ts

```diff
@@ -1,5 +1,8 @@
-import {resolve, relative} from 'path';
+import {resolve, relative, dirname} from 'path';
+import {sync as glob} from 'glob';
+import {statSync as stat} from 'fs';
 import {createHash} from 'crypto';
+import {readFile, writeFile, mkdirp} from 'fs-extra';
```

Comment:
Are we able to leverage the [`FileSystem` class](https://github.com/lemonmade/sewing-kit/blob/main/packages/model/src/base/fs.ts) instead? It should be accessible through `pkg.fs`.

I haven't confirmed this yet with @lemonmade, but I believe the intention is to always use the `FileSystem` class whenever possible. I could be wrong here but I believe its so we can have a better e2e testing solution. We'll just have to mock the `Filesystem` class instead of a virtual filesystem.

Reply:
I think in the case of `pkg.fs` it's scoped to `pkg.root` (e.g. `<workspace_root>/packages/plugin-javascript`), which makes it difficult to do cache-related stuff in `.sewing-kit/cache`.

That said I could just create a new `FileSystem` instance for the cache 👍